### PR TITLE
[4.0] Namespace com_plugins

### PR DESCRIPTION
**Pull request by joomdonation**
Pull Request for Issue # .

### Summary of Changes
This simple PR convert com_plugins component to namespace MVC for Joomla 4.

### Testing Instructions
1. Install latest Joomla 4.0-dev with sample data, then apply patch (or better install my branch https://github.com/joomdonation/joomla-cms/archive/namespace_com_plugins.zip )

2. Access to Extensions -> Plugins, try to update, enable, disable , checkin plugins, make sure it is working as expected

### CODE REVIEW

File: administrator/components/com_plugins/Controller/Controller.php [hunk 1]
```diff
@@ -6,30 +6,40 @@
  * @copyright   Copyright (C) 2005 - 2017 Open Source Matters, Inc. All rights reserved.
  * @license     GNU General Public License version 2 or later; see LICENSE.txt
  */
+namespace Joomla\Component\Plugins\Administrator\Controller;
 
 defined('_JEXEC') or die;
 
+use Joomla\CMS\Controller\Controller as BasseController;
```
Comment (Quy) [inline administrator/components/com_plugins/Controller/Controller.php]: Typo `Basse`.

File: administrator/components/com_plugins/Controller/Controller.php [hunk 2]
```diff
@@ -6,30 +6,40 @@
...
+use Joomla\Component\Plugins\Administrator\Helper\PluginsHelper;
+
 /**
  * Plugins master display controller.
  *
  * @since  1.5
  */
-class PluginsController extends JControllerLegacy
+class Controller extends BasseController
```
Comment (Quy) [inline administrator/components/com_plugins/Controller/Controller.php]: Typo `Basse`.

**Comment by joomdonation** [comment]
drone error is unclear reason and un-related to the change in this PR

File: administrator/components/com_plugins/View/Plugins/Html.php [hunk 3]
```diff
@@ -6,15 +6,19 @@
...
+use Joomla\CMS\Helper\ContentHelper;
+use Joomla\CMS\View\HtmlView;
+
 /**
  * View class for a list of plugins.
  *
  * @since  1.5
  */
-class PluginsViewPlugins extends JViewLegacy
+class Html extends HtmlView
```
Comment (brianteeman) [inline administrator/components/com_plugins/View/Plugins/Html.php]: is this correct?

File: administrator/components/com_plugins/dispatcher.php [hunk 4]
```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_content
```
Comment (brianteeman) [inline administrator/components/com_plugins/dispatcher.php]: this should be com_plugins

File: administrator/components/com_plugins/View/Plugins/Html.php (hunk 3 above)
Comment (joomdonation) [inline administrator/components/com_plugins/View/Plugins/Html.php]: Yes. This is correct

**Comment by wilsonge** [comment]
Thankyou :)

**Pull request merged**